### PR TITLE
#159965310 mockups for article likes and dislikes

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
#### what does this pull request do
- Adds mockups for article likes and dislikes

#### Description of tasks to be completed
- Have buttons alongside an article for a user to click to like or dislike the article 
- Show the number of likes and dislikes for a given article
- have a navigation bar for different sections

#### How should this be tested manually
- Clone this repository
- Run the command `git checkout ft-likes-dislikes-mockups-159965310`
- change directory to `ui-mockups` by executing `cd ui-mockups` in your terminal
- Do the following commands to set your environment for testing using your terminal
    - Install [node](Download | Node.js)
    - Run `npm install` to install all packages defined in `package.json` file
    - npm install gulp-cli -g
    - npm install gulp -D
- After executing above commands,type `gulp` and hit `[Enter]` button
- On your browser, visit the follwing pages
    - http://localhost:3000/articles.html
    - http://localhost:3000/article.html
    - http://localhost:3000/myarticles.html
#### Any background context you want to provide?
Having mockups is important in every API development because it helps to visualize how the user interface will look like in the finished product.

#### What are the relevant pivotal tracker stories?
[Finishes #159965310]

#### screenshots
image

### CODE REVIEW

**Comment by mikenthiwa** [comment]
@meshack-mbuvi The button is not visible.
I noticed that the space betwwen like and dislike button is small increase the margin between them in article.html.
Also remove the background image.

File: ui-mockups/src/articles.html [hunk 1]
```diff
@@ -87,20 +119,24 @@ <h6><a href="article.html">Donec id elit non mi porta</a></h6>
                                 </small>
                             </p>
                             <small>
-                                <a class="ah-author-link" href="#">Meshack Mbuvi</a>
+                                By <a class="ah-author-link" href="#">Mbuvi</a>
                             </small>
-                            <p class="d-flex justify-content-between">
+                            <p class="d-flex justify-content-between article-footer">
```
Comment (pabishai) [inline ui-mockups/src/articles.html]: @meshack-mbuvi Is the lower border necessary. Also I think the colors do not really blend into the existing template we have.

**Review by pabishai — CHANGES_REQUESTED** [review]
Reviewed your PR

File: ui-mockups/src/articles.html (hunk 1 above)
Comment (meshack-mbuvi) [reply ui-mockups/src/articles.html]: Which color do you recommend for the likes and dislikes @PAbishai ?

**Comment by micahoriaso** [comment]
@meshack-mbuvi please remove all underlines from under the articles and the underline under the likes and dislikes buttons. Also, remove the background hover on the likes and dislikes button instead change the colour of the icons themselves. All the undelines make the articles page look crowded, therefore giving the user too much information making the user distracted from the main point which is reading the article.

**Comment by meshack-mbuvi** [comment]
@micahoriaso Give reasons why I should do so

File: ui-mockups/src/articles.html (hunk 1 above)
Comment (pabishai) [reply ui-mockups/src/articles.html]: Maybe space the icons and the number of likes a bit and tint the color a bit to make it lighter and blend in more with the other colors

**Comment by Bileonaire** [comment]
@meshack-mbuvi good work.
I suggest that you remove the wordings in the like and dislike buttons because the icons serve the purpose.

**Comment by meshack-mbuvi** [comment]
@Bileonaire I understand your concern, I got the implementation from facebook. Was thinking that some users may not get the full meaning for a thumb up without the words.Am considering your feedback.

**Comment by pngg** [comment]
Looks good!

**Comment by micahoriaso** [comment]
Looks good to me.

**Comment by micahoriaso** [comment]
> @meshack-mbuvi please remove all underlines from under the articles and the underline under the likes and dislikes buttons. Also, remove the background hover on the likes and dislikes button instead change the colour of the icons themselves. All the undelines make the articles page look crowded, therefore giving the user too much information making the user distracted from the main point which is reading the article.

Looks good to me

**Review by pabishai — APPROVED** [review]

**Pull request merged**